pkg/interactor/save_data_test.go

```diff
@@ -0,0 +1,95 @@
+package interactor
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	// ErrSaveDataDB is returned when the database fails to save data
+	ErrSaveDataDB = errors.New("error when trying to save data")
+
+	// ErrTokenEmpty is returned for empty access tokens.
+	ErrTokenEmpty = errors.New("no access token provided")
+
+	FakeDate, _ = time.Parse("2006-01-02 15:04:05", "2020-04-04 00:00:00")
```

Comment:
Do you mind adding a comment as to what `FakeDate` represents? Also, is it used elsewhere? If not, here's a suggestion 👇 

 ```suggestion
	// fakeDate represents a valid date when saving lala
	fakeDate, _ = time.Parse("2006-01-02 15:04:05", "2020-04-04 00:00:00")
```

Reply:
Done!